doc/7/controllers/document/m-upsert/snippets/m-upsert.js

```diff
@@ -0,0 +1,45 @@
+const doc1 = { capacity: 4 };
+const doc2 = { capacity: 7 };
+
+try {
+  await kuzzle.document.create('nyc-open-data', 'yellow-taxi', doc1, 'some-id');
+  await kuzzle.document.create('nyc-open-data', 'yellow-taxi', doc2, 'some-other-id');
+
+  const documents = [
+    {
+      _id: 'some-id',
+      changes: { category: 'sedan' }
+    },
+    {
+      _id: 'some-other-id',
+      changes: { category: 'limousine' }
+    }
+  ];
```

Comment:
(nitpicking) It would be nice to create a new document on the fly, with default value(s)